Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -246,16 +246,14 @@ private IList<Parameter> CreateQueryParameters(bool isCompositeContext)
 
         private Operation CreatePutByIdOperation(OpenApiMetadataPathsResource openApiMetadataResource)
         {
-            return new Operation
+            var op = new Operation
```

Comment:
After refactoring the logic (as described below), revert this line and return the operation directly.

Reply:
updated